### DISCUSSION TITLE
librespot output as internet radio stream

### DISCUSSION THREAD

**Discussion by TheFrozenDuck** [librespot-org/librespot#833] [Feature Requests]
I got the odd case of needing the output of librespot available as internet radio streams. I looked into icecast but it seems to be quite a lot of overhead for a single audio stream. I also found no other solutions yet and so I was wondering if anyone already had a similiar problem and was able to solve it.

For the case that there is no solution I was wondering, if internet radio support in form of a new audio backend might be welcome.

**Comment by roderickvd** [comment]
Retransmitting audio over a network is forbidden under the Spotify ToS, and so is not something I would want to have.
